### ISSUE TITLE
Adding new users - very complex passwords are "too weak"

### ISSUE THREAD

**Issue by cbarratt** [Ubiquiti-App/UNMS#60]
I've been trying to add a secondary user to my UNMS install and no matter how long or how complex the password is it says it is too weak.

even passwords generated via 1Password's generator for e.g
`etjyep+HE73(ieCLpHrX`

Meaning I can't actually invite any further users.

**Comment by cbarratt** [comment]
I think I've tracked it down. I initially typed in "password" as a test to see what happened. Then proceeded to type in a more complex password. I saw these errors in the dev console - this is what seems to have caused it to block / report all passwords as insecure.

![screen shot 2017-08-01 at 09 57 30](https://user-images.githubusercontent.com/1702826/28817384-f444c584-769f-11e7-9733-ad24bdc63107.png)
